### PR TITLE
Support @Column("alias_name") annotation

### PR DESCRIPTION
**Pull request by moznion**
This annotation allows to specify the column name for each columns as you like.

### CODE REVIEW

**Comment by moznion** [comment]
By any chance, `@NamedColumn` is better than `@ColumnName`?

**Comment by tokuhirom** [comment]
why don't you use @Column(name="hoge")？

**Comment by moznion** [comment]
Ah, that is better!
I rewrote like so.

File: tinyorm/src/main/java/me/geso/tinyorm/annotations/Column.java [hunk 1]
```diff
@@ -8,5 +8,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 public @interface Column {
-
+	String value() default "";
```
Comment (tokuhirom) [inline tinyorm/src/main/java/me/geso/tinyorm/annotations/Column.java]: default should be null.

**Comment by moznion** [comment]
It is impossible I think...

**Pull request merged**